_docs/_latest/use/model.md

```diff
@@ -5,8 +5,8 @@ order: 004
 
 # {{ page.title }}
 
-Forseti models built on top of the raw inventory data to provide us with a better 
-understanding of how the data is related to each other.
+Forseti models are built on top of the raw inventory data to provide us with
```

Comment:
Simplify "... raw Inventory data to help show how the data is related."

Reply:
done